exchanges/bitmex/bitmex_websocket.go

```diff
@@ -131,338 +152,292 @@ func (b *Bitmex) wsReadData() {
 }
 
 func (b *Bitmex) wsHandleData(respRaw []byte) error {
-	quickCapture := make(map[string]interface{})
-	err := json.Unmarshal(respRaw, &quickCapture)
+	var err error
```

Comment:
All linter placations are distasteful when it comes to err; ( _But this is my one_ ).
This one is just the least intrusive way of dealing with `gocritic.sloppyReassign` and `govet.shadow` fighting over whether it's simultaneously right and wrong to shadow a var.